main.py

```diff
@@ -356,6 +405,25 @@ def __init__(self, *args):
 		self.actionExit.triggered.connect(self.onExitClicked)
 		self.actionSettings.triggered.connect(self.onSettingsClicked)
 
+		if not "search_for_updates" in self.settings.settings or not "job_id" in self.settings.settings:
+			self.settings.settings["search_for_updates"] = False
+			self.settings.settings["job_id"] = ""
+			self.settings.save()
+
+		if self.settings.settings["search_for_updates"] == True:
+			if os.name == "nt":
+				updater = Updater(self.settings)
+				if not updater.jobs[0]["jobId"] == updater.gsettings.settings["job_id"]:
+					reply = QMessageBox.question(self, 'Update', "A new update for XQEMU is available, would you like to update?\nThe size of the update is: " + str(round(updater.release[0]["size"] / 1e+6, 2)) + " MB", QMessageBox.Yes, QMessageBox.No)
```

Comment:
1e+6 should probably be more like `updater.release[0]["size"] / 1000 / 1000`; possibly even:

```
size_kilobytes = updater.release[0]["size"] / 1000
size_megabytes = size_kilobytes / 1000
```